ui/components/ui/error-message/index.scss

```diff
@@ -19,4 +19,17 @@
   &__text {
     overflow: auto;
   }
+
+  &__link {
+    text-decoration: none;
+    color: $Blue-500;
+
+    &:hover {
+      color: $Blue-400;
+    }
+
+    &:active {
+      color: $Blue-600;
+    }
+  }
```

Comment:
Since we moved to CSS variables, we need to change `$Blue-500` to `var(--Blue-500)`, as with the other two.